package.json

```diff
@@ -200,6 +200,7 @@
     "tslint": "6.1.3",
     "typescript": "4.2.4",
     "wait-on": "^5.3.0",
+    "webpack-bundle-analyzer": "^4.5.0",
```

Comment:
Avoid using `webpack-bundle-analyzer` as `source-map-explorer` is recommended by Angular team